src/charm.py

```diff
@@ -109,7 +109,21 @@ def _update_config(self) -> bool:
           True if config changed; False otherwise
         """
         alertmanagers = self.karma_consumer.get_alertmanager_servers()
-        config = {"alertmanager": {"servers": alertmanagers}, "listen": {"port": self.port}}
+
+        config = {
+            "alertmanager": {"servers": alertmanagers},
+            "listen": {
+                "port": self.port,
+                # "cors": {"allowedOrigins": [am["uri"] for am in alertmanagers]},
```

Comment:
@dparv I tried this and karma would parse this.
Using `proxy: true` worked just fine.
Please let me know what is your ideal deployment scenario.